packages/react-components/src/hooks/use-toast.ts

```diff
@@ -3,9 +3,10 @@
 import * as React from 'react';
 import { type ToastProps } from '../toast/toast';
 
-const TOAST_LIMIT = 1;
 const TOAST_REMOVE_DELAY = 10000;
 
+type Variant = 'default' | 'success' | 'error' | 'warning' | 'loading';
```

Comment:
Can you export this type @Pratosh22? Also maybe rename it to something like `ToasterVariants`